app/views/locales/_locale_form.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_for(@locale, html: { class: 'post' }) do |f| %>
+  <p class="required notice">* <%= t('.required_notice', :default => "Required information") %></p>
```

Comment:
Here, you're using t(), but in the app/views/preferences/index.html.erb file below, the strings are wrapped in ts(). We probably need to decide which we want to use going forward.